Index: pom.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- pom.xml	(revision 35a21d499357b464ebb5b571cb97ac0bc5e57f01)
+++ pom.xml	(revision 9838d0cefa7b1babfb52ff1702bcb490e934761c)
@@ -53,6 +53,14 @@
             <plugin>
                 <groupId>org.springframework.boot</groupId>
                 <artifactId>spring-boot-maven-plugin</artifactId>
+                <configuration>
+                    <excludes>
+                        <exclude>
+                            <groupId>org.projectlombok</groupId>
+                            <artifactId>lombok</artifactId>
+                        </exclude>
+                    </excludes>
+                </configuration>
             </plugin>
         </plugins>
     </build>
Index: src/main/java/ru/javaops/bootjava/model/Role.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/ru/javaops/bootjava/model/Role.java	(revision 9838d0cefa7b1babfb52ff1702bcb490e934761c)
+++ src/main/java/ru/javaops/bootjava/model/Role.java	(revision 9838d0cefa7b1babfb52ff1702bcb490e934761c)
@@ -0,0 +1,6 @@
+package ru.javaops.bootjava.model;
+
+public enum Role {
+    ROLE_USER,
+    ROLE_ADMIN
+}
\ No newline at end of file
Index: src/main/java/ru/javaops/bootjava/model/User.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/ru/javaops/bootjava/model/User.java	(revision 9838d0cefa7b1babfb52ff1702bcb490e934761c)
+++ src/main/java/ru/javaops/bootjava/model/User.java	(revision 9838d0cefa7b1babfb52ff1702bcb490e934761c)
@@ -0,0 +1,23 @@
+package ru.javaops.bootjava.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.Set;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class User {
+
+    private String email;
+
+    private String firstName;
+
+    private String lastName;
+
+    private String password;
+
+    private Set<Role> roles;
+}
\ No newline at end of file
